xarray/core/dask_array_ops.py

```diff
@@ -27,7 +28,109 @@ def dask_rolling_wrapper(moving_func, a, window, min_count=None, axis=-1):
     return result
 
 
-def rolling_window(a, axis, window, center, fill_value):
+def rolling_window(a, axis, window, center, fill_value, mode):
+    import dask
+
+    if version.LooseVersion(dask.__version__) < "1.7":
+        if mode is not None:
+            raise NotImplementedError(
+                "dask >= 1.7 is required for mode={}.".format(mode)
```

Comment:
Is this the right version number for dask? I can't find a 1.7 release.

Also note that we only need to support versions of dask < 6 months old, per http://xarray.pydata.org/en/stable/installing.html. If we can just delete the old code entirely, that might be cleaner.

Reply:
[Dask API page](https://docs.dask.org/en/latest/array-api.html#dask.array.pad) says it came at the version 1.7, but I just noticed this doc is just copied from numpy...
I'll fix it.